### PR TITLE
Fix reuters us full-text fetching

### LINKED ISSUE #7529: 路透社英文分站无法正常获取
#### 路由地址（不包含参数）

```routes
/reuters/channel/:site/:channel
```

#### 完整路由地址，包含所有必选与可选参数

```fullroutes
https://rsshub.app/reuters/channel/us/world
https://rsshub.app/reuters/channel/uk/world
```

#### 相关文档地址

https://docs.rsshub.app/traditional-media.html#lu-tou-she

#### 预期是什么

正常获取

#### 实际发生了什么

英文分站 (us, uk) 无法正常获取，cn 正常。

### PR DESCRIPTION
**Pull request by oppilate**
## 该 PR 相关 Issue / Involved issue

Close #7529 #7825

## 完整路由地址 / Example for the proposed route(s)

```routes
/reuters/channel/us/world
/reuters/channel/us/business
/reuters/channel/us/legal
/reuters/channel/us/markets
/reuters/channel/us/breakingviews
/reuters/channel/us/technology
/reuters/channel/us/investigates
/reuters/channel/us/lifestyle
```

## 新RSS检查列表 / New RSS Script Checklist

- [ ] 这个PR中包含了新的路由吗? Does this PR add new route?
  - 如果有, 请完成检查列表. If yes, please finish the check list
  - **如果你的PR符合下方某个事项, 也请注明. If any of the checklist item meets your PR, please fill it out.**
  - [x] <- 这样打勾
- [ ] 是否提供了文档? Documentation provided?
  - [ ] 是否提供了英文文档? EN Documentation provided?
- [x] 是否支持全文获取? Is this RSS Script support fulltext?
  - [ ] 如果全文获取中需要访问文章链接, 是否使用了缓存? If fulltext requires to fetch detail pages, is cache used in the process?
  - [缓存说明](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun) | [How to use cache](https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun)
- [ ] 目标是否有明显的反爬/频率限制? Is there any sign of anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? (延长缓存时间, 写文档说明, etc.) If yes, do your code reflect this sign? (e.g. write documentations, use long cache time)
- [ ] 目标是否有提供日期？ Is there a date in the source?
  - [ ] 如果有，包是否正确解析？ If there is, can this script provide this info?
  - [ ] 如果有提供解析能力，时区是否正确调整？ Is the timezone correctly provided?
  - 如果有提供日期，但是没有提供解析，请说明原因 If there is a date but this script does not parse, please provide your reason.
- [ ] 是否引入的新的包? Any new package introduced?
  - 如果有, 请说明原因. If yes, please state your reason
- [ ] 是否使用了`Puppeteer`? Make use of `Puppeteer`?
  - 如果有, 请说明原因. If yes, please state your reason
  

## 说明 / Note

Based on https://github.com/DIYgod/RSSHub/pull/7825. Fixed the full-text fetching.

### CODE REVIEW

**Review by SettingDust — CHANGES_REQUESTED** [review]
无法显示所有图片？只显示了幻灯第一张

File: lib/routes/reuters/utilsUS.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+const ProcessFeedUS = async (link) => {
+    const response = await got.get(link);
+    const $ = cheerio.load(response.data);
+    const pubDate = $('meta[name="article:published_time"]').attr('content');
```
Comment (TonyRL) [inline lib/routes/reuters/utilsUS.js:7]: Please use parseDate() from `@/utils/parse-date`

**Review by TonyRL — CHANGES_REQUESTED** [review]
Please use art-template for generating HTML elements <video>, <figure>, etc. 
See https://github.com/DIYgod/RSSHub/discussions/8359

**Comment by TonyRL** [comment]
Superseded by https://github.com/DIYgod/RSSHub/pull/9761.

**Pull request closed without merging**